src/ledger.js

```diff
@@ -0,0 +1,131 @@
+// LedgerSystem class
+const BigNumber = require('bignumber.js');
+const Data = require("./database");
+const FIFOQueue = require("./queue");
+const sleep = require('./utils').sleep;
+const groupBy = require('./utils').groupBy;
+const OperationTypes = {
+    TRANSFER: "TRANSFER"
+}
+const OperationStatus = {
+    INIT: "INIT",
+    PROCESSING: "PROCESSING",
+    APPLIED: "APPLIED",
+    REJECTED: "REJECTED"
+}
+const Entities = {
+    OPERATIONS: "operations",
```

Comment:
please make key and value same to avoid confusion - we've had a lot of it in coinswitch. 
if there is some proper Enum construct you can use, use that instead